mandatory/3-playing-computer.js

```diff
@@ -7,28 +7,42 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  A1: because b was not defined.
   2. Remove the line that throws the error.
+  A2: console.log(b) was deleted
   3. What is printed to the console?
+  A3: 
```

Comment:
This output is very close but not quite correct, I'd recommend running the file again 👍